frontend/src/store/group-creation/actions.ts

```diff
@@ -0,0 +1,25 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import {
+  EAMGroupCreateResponseDto,
+  EAMGroupCreateRequestDto,
+  AsyncThunkConfig,
+} from 'common/types/types';
+import { ActionType } from './common';
+
+const create = createAsyncThunk<
+  EAMGroupCreateResponseDto,
+  EAMGroupCreateRequestDto,
+  AsyncThunkConfig
+>(ActionType.CREATE, async (registerPayload, { extra }) => {
+  const { groupApi } = extra;
+
+  const tenantId = 'd94b2c68-289d-467c-914f-ababcdc8fcaa';
```

Comment:
Add `console.warn('Replace tenant id by actual value')` here to not forget about it in the future